src/Sylius/Bundle/CoreBundle/CHANGELOG.md

```diff
@@ -7,3 +7,7 @@ CHANGELOG
 * Twig extension was renamed from `SyliusRestrictedZoneExtension` into `RestrictedZoneExtension`,
   also the service name was changed from `sylius.twig.restricted_zone_extension`
   to `sylius.twig.extension.restricted_zone`.
+
+### v0.16.0
+
+* Change calculatorRegistry type to `ServiceRegistryInterface` in `ShippingMethodTypeSpec`
```

Comment:
I'd say ``Replaced custom RuleCheckerRegistry and CalculatorRegistry with ServiceRegistry component.``